webhook-subscription-2022.md

```diff
@@ -190,136 +178,80 @@ Content-Type: application/ld+json
 
 The webhook request `MUST` follow the standard outlined in the [Acivity Pub Update Type](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-update)
 
-#### Delete
+#### 4.1.2 Delete
 
 If a resource is deleted, the Pod `MUST` send a webhook request to all subscribed targets.
 
 The webhook request `MUST` follow the standard outlined in the [Acivity Pub Delete Type](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-delete)
 
-## Unsubscribe API
+## 5. Unsubscribe API
 
-All servers implementing the `WebHookSubscription2021` protocol `MUST` provide a unique resource for each subscription. This will be called the "Unsubscribe Endpoint."
+All servers implementing the `WebHookSubscription2022` protocol `MUST` provide a unique resource for each subscription. This will be called the "Unsubscribe Endpoint."
 
 If a `DELETE` request is received at the unsubscribe endpoint, the Pod `MUST` forget that subscription and any subsequent actions should not trigger a webhook request.
 
 If a request is received at the unsubscribe endpoint and it does not include an Authorization or DPoP header, the Authorization or DPoP Headers are invalid, or the WebId in the authorization header does not correspond with the WebId that made the subscription, the Pod `MUST` reject the request.
 
-## Features
-
+## 6. Features
 This section details the additional features built for `WebHookSubscription2021`.
 
-### webhook-auth
+### 6.1 webhook-auth
 The `webhook-auth` feature allows subscribing servers to verify that a request came from a certain Pod.
 
-Pods that implement the `webhook-auth` feature `MUST` include an HTTP URL that produces a JSON Web Key Set.
-
-Pods that implement the `webhook-auth` feature `SHOULD` rotate keys on its JSON Web Key Set.
-
-Pods that implement the `webhook-auth` feature `MUST` include the `jwks_endpoint` field in the Solid Metadata document. The value of this field is the Pod's JWKS URI.
-
-When a webhook is triggered, compliant Pods `MUST` include an `authorization` header with a token signed by a key in its JSON Web Key Set. The token `MUST` include the following fields:
- - exp: A number indicating the expiration of the token
- - htu: The url of the webhook target
- - htm: The method of the webhook request. Because all webhook request use the `POST` method, this value will always be `POST`.
+Pods that implement the `webhook-auth` feature `MUST` create an Access Token and DPoP token in accordance with the Solid OIDC specification and embed those tokens in the headers of all webhook requests.
 
 ## Further Considerations
+ - Using DPoP authentication in the webhook request does coincide with the common use case for Solid-OIDC, but it is overkill. There is no need for a security model that has two separate tokens (DPoP Token and the Auth token). Instead, we should consider using one token with the combined features (identity + htm and htu) of the two tokens.
```

Comment:
Should this rather be brought up as an independent issue in the authN panel? It seems more like commentary and arguably a bit out of place in a specification document.

Reply:
Perhaps. The proper way to address this is also defining a non-DPoP version of Solid OIDC so a client can use that.

Reply:
I think this section can stay and be used as a reference in the suggested AuthN panel issue. Once it gets addressed there this document can be updated.